owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/fragments/SettingsPictureUploadsFragment.kt

```diff
@@ -117,6 +120,10 @@ class SettingsPictureUploadsFragment : PreferenceFragmentCompat() {
                 prefPictureUploadsPath?.summary = DisplayUtils.getPathWithoutLastSlash(it.uploadPath)
                 prefPictureUploadsSourcePath?.summary = DisplayUtils.getPathWithoutLastSlash(it.sourcePath.toUri().path)
                 prefPictureUploadsOnWifi?.isChecked = it.wifiOnly
+                prefPictureUploadsOnCharging?.apply {
+                    isVisible = it.wifiOnly
```

Comment:
When I implemented the auto upload pictures and videos sections, we had a quick meeting and we decided to keep items not **enabled** instead of not **visible** when they cannot be activated yet, so that user perceives the app's value and all its possibilities since the beginning and not by discovering them